sdcm/cluster.py

```diff
@@ -3191,12 +3175,31 @@ def check_cluster_health(self):
             self.log.debug('Cluster health check disabled')
             return
 
-        for node in self.nodes:
-            node.check_node_health()
+        nemesis_count = len(self.nemesis)
```

Comment:
what is returned in case of parallel nemesis?

Reply:
Added the INFO that in case of parallel nemesis nodes health check is not run